src/components/EmailIdentityForm.tsx

```diff
@@ -3,7 +3,9 @@ import styled from "styled-components";
 
 import { IdentityTypes } from "../@types/Identity";
 
-export const InputForm: React.FC<{ type: IdentityTypes }> = ({ type }) => {
+export const EmailIdentityForm: React.FC<{ type: IdentityTypes }> = ({
+  type,
+}) => {
   const [formValue, setFormValue] = React.useState("");
```

Comment:
Well, if you made the change here, what about 

`const [email, setEmail] = React.useState("")` ? 😇

Reply:
Woops 🙇

Reply:
Wait, actually, this is also used to add/remove phone number 🤦‍♂️. Sorry, I'm not awake.

```ts
<Flex>
            <Button
              onClick={() => setAddPhone(!addEmail)}
              color={theme.colors.green}
            >
              Add a phone number
            </Button>
            {addPhone && <EmailIdentityForm type={IdentityTypes.PHONE} />}
          </Flex>
```

Reply:
f6daa83